coordinator/persistence-api/src/main/java/org/apache/cassandra/stargate/exceptions/RequestFailureReason.java

```diff
@@ -93,16 +97,16 @@ public enum RequestFailureReason {
   BACKUP_SERVICE_NOT_RUNNING,
 
   /** The file could not be uploaded or downloaded from remote storage. */
-  REMOTE_STORAGE_FAILURE,
+  REMOTE_STORAGE_FAILURE(502),
 
   /** The node is still bootstrapping and is therefore not ready to serve read requests. */
   BOOTSTRAPPING,
 
   /** The read size limit has been exceeded. */
-  READ_SIZE,
+  READ_SIZE(4),
 
   /** Couldn't complete query (typically paxos transaction) due to down node. */
-  NODE_DOWN,
+  NODE_DOWN(5),
```

Comment:
should we check in `fromCode` if `codeToReasonMap[code]` is `null`?

Reply:
I need to update the stargate patch to the newest changes from datastax/cassandra. I changed there `fromCode` to HashMap. I will bring things here once the datastax/cassandra CI confirm I did not mess up anything with my latest changes.